app/src/main/res/layout/fragment_product_details.xml

```diff
@@ -1,176 +1,174 @@
+        <ImageView
+            android:id="@+id/mainImage"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:paddingTop="100dp"
+            android:scaleType="centerCrop"
+            android:src="@{product.imageURL}"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:src="@drawable/photomodel" />
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/content_root"
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:paddingBottom="112dp"
-            android:background="@color/background_color">
+            android:layout_height="0dp"
+            android:background="@color/background_color"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/mainImage"
+            tools:context=".presentation.activity.MainActivity">
+
+
+            <ToggleButton
+                android:id="@+id/favorite"
+                android:layout_width="@dimen/favorite_button_size"
+                android:layout_height="@dimen/favorite_button_size"
+                android:layout_marginTop="@dimen/dimen_default_margin"
+                android:layout_marginEnd="@dimen/dimen_default_margin"
+                android:background="@drawable/selector_favorite_button"
+                android:checked="@{viewModel.favoriteIsChecked}"
+                android:elevation="20dp"
+                android:onClick="@{()->viewModel.favoriteClicked()}"
+                android:textOff=""
+                android:textOn=""
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/price"
+                android:layout_width="wrap_content"
+                android:layout_height="@dimen/dimen_headers_height"
+                android:layout_marginTop="24dp"
+                android:layout_marginEnd="@dimen/dimen_default_margin"
+                android:fontFamily="@font/metropolis_bold"
+                android:text='@{product.price + "$"}'
+                android:textSize="@dimen/dimen_headers_text_size"
+                android:textStyle="bold"
+                android:visibility="@{product.price != null ?View.VISIBLE : View.INVISIBLE}"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/favorite"
+                tools:text="@string/prise" />
+
+            <androidx.appcompat.widget.AppCompatTextView
+                android:id="@+id/brand"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/dimen_default_margin"
+                android:ellipsize="end"
+                android:maxWidth="180dp"
```

Comment:
Please remove maxWidth, ellipsize, singleLine,maxLines from this view. It's required to allow user to see full product name on the pdp page. Allign this view like in figma, from start of the screen to the start of the price field

Reply:
fixed